cli/sawtooth_cli/submit.py

```diff
@@ -34,16 +35,23 @@ def _split_batch_list(args, batch_list):
 
 
 def do_submit(args):
-    with open(args.filename, mode='rb') as fd:
-        batches = batch_pb2.BatchList()
-        batches.ParseFromString(fd.read())
+
+    try:
+        with open(args.filename, mode='rb') as fd:
+            batches = batch_pb2.BatchList()
+            batches.ParseFromString(fd.read())
+    except FileNotFoundError as e:
+        raise CliException(e)
 
     rest_client = RestClient(args.url)
 
     start = time.time()
 
     for batch_list in _split_batch_list(args, batches):
-        rest_client.send_batches(batch_list)
+        try:
+            rest_client.send_batches(batch_list)
```

Comment:
Right. Testing, though, it worked better to have the exception passed up. Will chat on slack.